src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -162,6 +198,18 @@ public OperationResultResponse<ProjectInfo> Execute(ProjectRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
+            var existUsers = CheckUserExistence(request.Users.Select(u => u.UserId).ToList(), response.Errors);
+            if (!response.Errors.Any()
+                && existUsers.Count() != request.Users.Count())
+            {
+                response.Errors.Add("Not all users exist.");
```

Comment:
writte non exist users to errors

Reply:
we discussed it with Spartak, it's not necessary